dist/selleckt-legacy.js

```diff
@@ -25,22 +25,28 @@ module.exports = KEY_CODES;
 */
 'use strict';
 
-var MicroEvent  = function(){};
+var MicroEvent = function(){};
 MicroEvent.prototype = {
-    bind    : function(event, fct){
+    on: function(event, fct){
         this._events = this._events || {};
-        this._events[event] = this._events[event]   || [];
+        this._events[event] = this._events[event] || [];
         this._events[event].push(fct);
     },
-    unbind  : function(event, fct){
+    bind: this.on,
```

Comment:
`this` in this context isn't `MicroEvent`, it'll be the global `this` I think - `window`?